src/js/angular/models/graphql/graphql-endpoints-info.js

```diff
@@ -0,0 +1,139 @@
+export class GraphqlEndpointInfo {
+    constructor(data) {
+        this._id = data.id;
+        this._endpointId = data.endpointId;
+        this._endpointURI = data.endpointURI;
+        this._label = data.label;
+        this._description = data.description;
+        this._default = data.default;
+        this._active = data.active;
+        this._lastModified = data.lastModified;
+        this._objectsCount = data.objectsCount;
+        this._propertiesCount = data.propertiesCount;
+        this._warnings = data.warnings;
+        this._errors = data.errors;
+        this._status = data.status;
+    }
+
+    get id() {
+        return this._id;
+    }
+
+    set id(value) {
```

Comment:
Do we actually need setters?

Reply:
For setting values to "private" fields you need setters. Do you see some issue?